openvdb/openvdb/CMakeLists.txt

```diff
@@ -324,6 +324,7 @@ if(USE_EXPLICIT_INSTANTIATION)
 
   set(OPENVDB_LIBRARY_INSTANTIATION_ALL_TREES
     MaskTree
+    HalfTree
```

Comment:
Is this what you are thinking about, @danrbailey? @ghurstunither for viz.

Reply:
Yep, thanks!